SPECS/kata-containers/kata-containers.spec

```
@@ -1,222 +1,116 @@
-%{_prefix}/local/bin/kata-monitor
-%{_prefix}/local/bin/kata-runtime
-%dir %{_datadir}/defaults/kata-containers/
-%{_datadir}/defaults/kata-containers/configuration*.toml
-%{_datadir}/bash-completion/completions/kata-runtime
 %license LICENSE
 %doc CONTRIBUTING.md
 %doc README.md
 
-# CRI-O drop-in file
-%{_sysconfdir}/crio/crio.conf.d/50-kata
-
 %files tools
-# osbuilddir
-%dir %{kataosbuilderdir}
-%dir %{katalocalstatecachedir}
-%{kataosbuilderdir}/*
-
-# agent
-%dir %{kataagentdir}
-%{kataagentdir}/*
-
-%dir %{katauvmdir}
-%{katauvmdir}/VERSION
-%{katauvmdir}/versions.yaml
-%{katauvmdir}/mariner-build-uvm.sh
-%{katauvmdir}/ci/install_yq.sh
-
-# Remove some scripts we don't use
-%exclude %{kataosbuilderdir}/rootfs-builder/alpine
-%exclude %{kataosbuilderdir}/rootfs-builder/centos
-%exclude %{kataosbuilderdir}/rootfs-builder/clearlinux
-%exclude %{kataosbuilderdir}/rootfs-builder/debian
-%exclude %{kataosbuilderdir}/rootfs-builder/template
-%exclude %{kataosbuilderdir}/rootfs-builder/ubuntu
+%dir %{kata_path}
+%dir %{tools_pkg}
+%dir %{tools_pkg}/tools
+%dir %{tools_pkg}/tools/osbuilder
+%{tools_pkg}/tools/osbuilder/Makefile
+
+%dir %{tools_pkg}/tools/osbuilder/scripts
+%{tools_pkg}/tools/osbuilder/scripts/lib.sh
+
+%dir %{tools_pkg}/tools/osbuilder/rootfs-builder
+%{tools_pkg}/tools/osbuilder/rootfs-builder/rootfs.sh
+%dir %{tools_pkg}/tools/osbuilder/rootfs-builder/cbl-mariner
+%{tools_pkg}/tools/osbuilder/rootfs-builder/cbl-mariner/config.sh
+%{tools_pkg}/tools/osbuilder/rootfs-builder/cbl-mariner/rootfs_lib.sh
+
+%dir %{tools_pkg}/tools/osbuilder/image-builder
+%{tools_pkg}/tools/osbuilder/image-builder/image_builder.sh
+
+%dir %{tools_pkg}/tools/osbuilder/node-builder
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/Makefile
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/clean.sh
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/common.sh
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/uvm_build.sh
+
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/bin
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/bin/kata-agent
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd
+%dir %{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd/system
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd/system/kata-containers.target
+%{tools_pkg}/tools/osbuilder/node-builder/azure-linux/agent-install/usr/lib/systemd/system/kata-agent.service
 
 %changelog
+* Fri Sep 20 2024 Manuel Huber <mahuber@microsoft.com> - 3.2.0.azl3-1
+- Upgrade to 3.2.0.azl3 release, using build instructions from fork
```

```suggestion
- Upgrade to 3.2.0.azl3 release, refactor build instructions
```
No need to reference the sources as a fork. From the packaging point of view, you are packaging the sources from the microsoft/kata-containers repository